templates/style/_navbar.scss

```diff
@@ -131,38 +131,86 @@ div.nav-container {
         }
     }
 
+    // These hardcoded values are "magic", in the sense they were observed to be the lowest-possible
+    // thresholds through experimentation with a package name that hit the max-width of the element.
+    $full-width: 1142px;
+    $medium-width: 994px;
+    $narrow-width: 901px;
+
     // use a .title span inside a menu to hide it on small screens
     span.title {
         display: none;
-        @media screen and (min-width: 872px) {
+        @media screen and (min-width: #{$narrow-width}) {
             display: inline;
         }
     }
 
     .pure-menu-right span.title {
-        @media screen and (min-width: 872px) {
-            display: none;
-        }
-        @media screen and (min-width: 965px) {
+        display: none;
+        @media screen and (min-width: #{$medium-width}) {
             display: inline;
         }
     }
 
-    form.landing-search-form-nav.not-latest {
-        span.title {
+     /// Generate the rules for the navbar menu item labels at various dimensions.
```

Comment:
That's a bit weird to see `/// ` used in a scss file. Is it a typo or a feature of scss? (and if it's a feature, what does it do?)

Reply:
[It's a Sass feature - it supports JSDoc-style comments.](https://sass-lang.com/documentation/syntax/comments#documentation-comments)

Reply:
Oh ok. Not sure if it'll be useful here but I guess it doesn't hurt.